lib/services/provisioningstates.go

```diff
@@ -33,12 +33,16 @@ type DownstreamID string
 // disambiguated by a DownstreamID when queried or updated.
 type ProvisioningStateID string
 
-// DownstreamProvisioningStates defines an interface for managing principal
-// provisioning state records scoped by a target downstream receiver.
-type DownstreamProvisioningStates interface {
+type DownstreamProvisioningStateGetter interface {
```

Comment:
Is this going to be used later?

Reply:
Yes, This interface will be use in e https://github.com/gravitational/teleport.e/pull/5191